assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -31,6 +31,14 @@ const DEFAULT_ELEMENT_WIDTH = PAGE_WIDTH / 2;
 // @todo Once none of the elements are placed randomly, default x can be moved to text/index.js
 const DEFAULT_LEFT_MARGIN = 40;
 
+const DEFAULT_TEXT_BORDER_RADIUS = {
```

Comment:
So my main impetus for making it a default value vs something that gets applied when changing to `Fill | Highlight` is if the user has a border radius applied, or explicitly set it to `0` and switched it to `Fill | Highlight`, it would keep getting overwriting the user input and seem cumbersome vs just having an initial suggested value that gets overwritten once the user sets it to their liking

Reply:
Yep, makes sense.